umbraco/Umbraco.CMS.Backoffice
# Feature: Sort children Entity Action

> [!NOTE]  
> Build on top of https://github.com/umbraco/Umbraco.CMS.Backoffice/pull/1391 . Please make sure it is merged first.



## Description

The PR adds the option to sort documents.

https://github.com/umbraco/Umbraco.CMS.Backoffice/assets/6078361/4e1ea61f-353f-4565-ba6f-5e8d25cd52a5

What to test:
* Create some documents in the root.
* Click on the entity action to sort the document
* Make sure the documents are in the correct order after the sorting is done.
* Do the same for some nested documents.
